src/main/java/edu/ucar/unidata/rosetta/service/FileParserManagerImpl.java

```diff
@@ -78,42 +85,29 @@ public String parseByLine(String filePath) {
      * used to parse the file data when both the header lines and delimiter(s)
      * have been specified by the user. TODO: refactor to return JSON
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)